### ISSUE TITLE
New internet service-ish error

### ISSUE THREAD

**Issue by tdtooke** [dhruv8sh/arch-update-checker#4]
Fairly recently this happened:

Error code: 0
Error:error: Unable to load summary from remote nuvola: While fetching https://dl.tiliado.eu/flatpak/nuvola/summary.idx: [6] Couldn't resolve host name
error: Invalid id : Name can't be empty
sed: -e expression #1, char 0: no previous regular expression

My thoughts are that this is not really a thing you can fix, just a remote service that is down.  But just so you know.  It was working fine earlier.

**Comment by dhruv8sh** [comment]
Was this broken only after the last update? And does the issue persist when you run the same command using a terminal?

**Comment by tdtooke** [comment]
Normally via terminal I'd type "pacaur -Syu" to update and do "flatpak update" to do the flatpaks.  No errors there.  From what I can tell the plasmoid still works it's just showing that error on the display.  It's also possible I may have done this to myself.  I'm troubleshooting a completely unrelated problem and I cleared a lot of my python cache files.  I want to say that it did work immediately after the update but at some point that happened.  On my own investigating I did find that tilliado.eu is down right now.
